conflog.tex

```diff
@@ -147,7 +147,7 @@ \subsection{Hotels and Accommodations}
 	\caption{Marriott Rooms: Left, a queen double. Right, a king single}		
 \end{figure} 
 
-UIUC is one of most well represented schools at past student conferences. Students from UIUC will be able to stay in their respective dorms and apartments to maximize the number of hotel rooms available to out of town guests. 
+UIUC is one of most well represented schools at past student conferences. Students from UIUC will be able to stay in their respective dorms and apartments to maximize the number of hotel rooms available to out of town guests. Sections seeking travel assistance are required to use the room blocks described above to prevent extra costs. Another proposed method to alleviate costs is to allow an option for students from smaller sections to be housed at an ANS-UIUC section member's apartment. The Diversity co-chair will assume responsibilities of reaching out to members-at-large and other minority serving institutions. 
```

Comment:
```suggestion
UIUC is one of most well represented schools at past student conferences. Students from UIUC will be able to stay in their respective dorms and apartments to maximize the number of hotel rooms available to out of town guests. Sections seeking travel assistance are required to use the room blocks described above to prevent extra costs. Another proposed method to alleviate costs is to allow an option for students from smaller sections to be housed at an ANS-UIUC section member's apartment. The Diversity co-chair will assume responsibilities of reaching out to members-at-large and minority serving institutions. 
```